osu.Game/Rulesets/Mods/IHasSeed.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.UserInterface;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Overlays.Settings;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public interface IHasSeed
+    {
+        Bindable<int?> Seed { get; }
+    }
+
+    public class SeedSettingsControl : SettingsItem<int?>
```

Comment:
@smoogipoo okay if i move this to its own file?

Reply:
Go for it